### PR TITLE
[ch3011] Fix button visual and text and Canary part of the feature

### PR DESCRIPTION
**Pull request by cacahouwete**
Add a few visual and language tweaks to the notification page
Add a canary

https://app.clubhouse.io/enmarche/story/3011/am%C3%A9liorations-de-la-gestion-des-notifications

### CODE REVIEW

File: front/style/pages/_profile.scss [hunk 1]
```diff
@@ -134,3 +134,9 @@
         }
     }
 }
+
+#adherent_email_subscription_emails_subscriptions {
```
Comment (marineb) [inline front/style/pages/_profile.scss]: We don't generally use IDs in CSS. Need to use a class. Also if it's just a margin, you can use one of our reusable classes.

**Comment by marineb** [comment]
@cacahouwete we should put all this in canary mode for now since it's not connected to anything. Also I couldn't find where you're infusing the `h3`. Maybe you can tell me on Monday. Thank you!

File: templates/user/set_email_notifications.html.twig [hunk 2]
```diff
@@ -2,20 +2,30 @@
...
-
         <div class="b__nudge--top">
             {{ form_widget(form.submit, { attr: { class: 'btn--blue' } }) }}
         </div>
 
+        <p class="text--small">
+            Pour ne plus recevoir les notifications des <strong>animateurs de comités</strong> ou de <strong>porteurs de projets citoyens</strong>, rendez-vous sur leurs pages respectives et arretez de les suivre.
+        </p>
+
+
```
Comment (illusionOfParadise) [inline templates/user/set_email_notifications.html.twig]: An extra line

File: src/Entity/Adherent.php [hunk 3]
```diff
@@ -452,20 +452,45 @@ public function getEmailsSubscriptions(): array
         if ($this->hasCitizenProjectCreationEmailSubscription()) {
             $subscriptions[] = AdherentEmailSubscription::SUBSCRIBED_EMAILS_CITIZEN_PROJECT_CREATION;
         }
+//      CANARY !
+        if ($this->localHostEmailsSubscription) {
+            $subscriptions[] = AdherentEmailSubscription::SUBSCRIBED_EMAILS_LOCAL_HOST;
```
Comment (ottaviano) [inline src/Entity/Adherent.php]: `LOCAL_HOST` => `LOCALHOST` :trollface:

Comment (cacahouwete) [reply src/Entity/Adherent.php]: :D I know ^^, it's one of the old fields removed ^^,

**Comment by thomasbisignani** [comment]
why don't using the `CanaryControllerTrait` to disable code in production ?

**Pull request merged**